src/app/_components/Card/CardSpeсialist/CardSpecialistExtended.js

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import P from 'prop-types';
```

Comment:
**P**?
consider using the PropTypes name like in other components

Reply:
Thanks for comment! Renamed.

Reply:
Done, thanks